### PR TITLE
docs: add info on publish modules

### PR DESCRIPTION
**Pull request by dhmlau**
Signed-off-by: Diana Lau <dhmlau@ca.ibm.com>

### CODE REVIEW

File: docs/publish-modules.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+## Publish other repos
+
+Pre-requisite: install [`strong-tools`](https://www.npmjs.com/package/strong-tools), we'll be using the `slt-release` command to publish.
+
+Here are the steps to publish:
+
+1. git clone the repo
+2. log into npmjs using `npm login` command
+3. run `slt-release -up <major/minor/patch>`. To determine which of major/minor/patch to choose, see https://semver.org/
+4. after the release, you can run `npm view <module-name>` to check.
```
Comment (achrinza) [inline docs/publish-modules.md:60]: We may also want to inform users to `npm logout` after publishing as I don't think the logged-in credentials would be necessary outside publishing.

```suggestion
4. after the release, you can run `npm view <module-name>` to check.

## Logging out after publishing

As best practice, logout of NPM using `npm logout` to mitigate against accidental publishing and token stealing.
```

**Review by achrinza — COMMENTED** [review]
Thanks for initiating this, @dhmlau!

I've added some comments below :point_down: 

Also, are we going to remove https://github.com/loopbackio/loopback-next/blob/00a62f630c19341ce399cec3a45563b1ead6e3b8/docs/site/DEVELOPING.md#releasing-new-versions, and point readers here instead?

File: docs/publish-modules.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+4. Post an announcement on [#general Slack channel](https://loopbackio.slack.com/archives/CE8CRBBFU).
+
+### Release cadence
+
+Roughly monthly release at the middle of the month. 
+
+### Troubleshooting
+
+1. When there are a subset of modules being published, what should I do to publish the rest?
```
[the remark this answers is not in the document]
Comment (dhmlau) [reply docs/publish-modules.md:44]: I'm not sure. I don't have any publish issue recently. For the Nov publish, there's the "service unavailable" error which I have no idea where it's coming from (redoing it works). Some time ago, I recalled that Raymond had timeout issues which I didn't see it.

File: docs/publish-modules.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+3. Run the publish command
+
+    In the `loopback-next` folder, run the following commands:
+    ```
+    $ npm i 
+    $ npm run publish
+    ```
+
+4. Post an announcement on [#general Slack channel](https://loopbackio.slack.com/archives/CE8CRBBFU).
```
[the remark this answers is not in the document]
Comment (dhmlau) [reply docs/publish-modules.md:36]: The `general` channel is one of the default channels when someone joins the LoopBack Slack workspace.
Only Workspace admin can post messages. It just reminded me to add the TSC member as workspace admin.

**Comment by dhmlau** [comment]
> Also, are we going to remove https://github.com/loopbackio/loopback-next/blob/00a62f630c19341ce399cec3a45563b1ead6e3b8/docs/site/DEVELOPING.md#releasing-new-versions, and point readers here instead?

It would be good. This md file has the instructions in publishing modules other than the loopback-next ones. So it fits better in this repo, IMHO.

**Review by achrinza — APPROVED** [review]

**Pull request merged**